compiler/rustc_interface/src/interface.rs

```diff
@@ -58,6 +58,11 @@ impl Compiler {
     }
 }
 
+#[allow(rustc::bad_opt_access)]
+pub fn set_parallel_mode(sopts: &config::UnstableOptions) {
+    rustc_data_structures::sync::set(sopts.threads > 1);
+}
```

Comment:
I can see where this function is called within rustdoc (in `src/librustdoc/lib.rs`), but I can't see where it is called in rustc. Am I overlooking something?

Reply:
it's in `run_compiler` in `rustc_driver_impl::lib.rs`

Reply:
In this PR, this part of the modification may be discarded because of rebease